src/mon/ConfigMonitor.cc

```diff
@@ -521,11 +521,9 @@ bool ConfigMonitor::prepare_command(MonOpRequestRef op)
     }
     goto update;
   } else if (prefix == "config reset") {
-    int64_t num = -1;
-    cmd_getval(g_ceph_context, cmdmap, "num", num);
-    int64_t revert_to = num;
-    if (revert_to < 0) // revert to last version
-      revert_to = version - 1;
+    int64_t revert_to = -1;
+    cmd_getval(g_ceph_context, cmdmap, "num", revert_to);
+    ceph_assert(revert_to >= 0);
```

Comment:
instead of assert here, return EINVAL? otherwise anyone can crash us from teh CLI, right?